src/content/docs/dropins/checkout/tutorials/multi-step.mdx

```diff
@@ -0,0 +1,1404 @@
+@media only screen and (width <= 768px) {
+  .checkout__wrapper {
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
+  }
+
+  .checkout__content {
+    grid-template-columns: 1fr;
+    gap: var(--spacing-big) 0;
+  }
+
+  .checkout__main,
+  .checkout__aside {
+    grid-column: auto;
+  }
+
+  .checkout__aside {
+    order: -1;
+  }
+}
+```
+
+<Diagram caption="Render containers with styles">
+![Render containers with styles](@images/dropins/checkout/render-containers-no-styles.png)
+</Diagram>
+</Task>
+
+<Task>
+### Display/remove empty cart
+
+Check if the cart is empty and display a message to the user. To do this, create a new container called `EmptyCart`.
+
+<Steps>
+1. Create two utility functions to display or remove the `EmptyCart` container when needed:
+
+   ```js
+   let emptyCart;
+   const displayEmptyCart = async () => {
+     if (emptyCart) return;
+
+     heading.setProps((prev) => ({
+       ...prev,
+       title: 'Empty Cart',
+     }));
+
+     emptyCart = await CartProvider.render(EmptyCart, {
+       routeCTA: () => '/',
+     })($emptyCart);
+
+     $content.classList.add('checkout__content--empty');
+   };
+
+   const removeEmptyCart = () => {
+     if (!emptyCart) return;
+
+     emptyCart.remove();
+     emptyCart = null;
+     $emptyCart.innerHTML = '';
+
+     heading.setProps((prev) => ({
+       ...prev,
+       title: 'Guest Checkout',
+     }));
+
+     $content.classList.remove('checkout__content--empty');
+   };
+   ```
+
+   <Aside type="note">
+   The example above also changes the heading title and adds a new class to the content wrapper so it can be hidden.
+   </Aside>
+
+1. Add the following declarations at the end of the **decorate** function to start listening for the `checkout/initialized` and `checkout/updated` events to determine when to display the empty cart.
+
+   ```js
+   events.on('checkout/initialized', handleCheckoutInitialized, { eager: true });
+   events.on('checkout/updated', handleCheckoutUpdated);
+   ```
+
+1. Add the missing handlers for the previous events.
+
+   ```js
+   async function handleCheckoutInitialized(data) {
+     if (isEmptyCart(data)) {
+       await displayEmptyCart();
+       return;
+     }
+   }
+
+   async function handleCheckoutUpdated(data) {
+     if (isEmptyCart(data)) {
+       await displayEmptyCart();
+       return;
+     }
+     removeEmptyCart();
+   }
+   ```
+
+   At this point, if you access the checkout page without a cart, you should see the `EmptyCart` container displayed.
+
+   <Diagram caption="Empty cart">
+   ![Empty cart](@images/dropins/checkout/empty-cart.png)
+   </Diagram>
+</Steps>
+</Task>
+
+<Task>
+### Handle shipping method form
+
+<Steps>
+1. You've probably noticed that even after initializing the checkout the shipping method form still loads. To resolve this, create a new utility method called `continueToShipping` to remove the skeleton and render the shipping method form when the checkout is initialized.
```

Comment:
I think this is not correct, the form we are rendering here is the shipping address form 👍🏻

Reply:
Good catch!